packages/cli/src/metrics/prometheus-metrics.service.ts

```diff
@@ -126,11 +137,25 @@ export class PrometheusMetricsService {
 	private mountMetricsEndpoint(app: express.Application) {
 		app.get('/metrics', async (_req: express.Request, res: express.Response) => {
 			const metrics = await promClient.register.metrics();
+			const prefixedMetrics = this.addPrefixToMetrics(metrics);
 			res.setHeader('Content-Type', promClient.register.contentType);
-			res.send(metrics).end();
+			res.send(prefixedMetrics).end();
 		});
 	}
 
+	private addPrefixToMetrics(metrics: string) {
```

Comment:
could we make this public and add unit tests just for this method?

Reply:
This is covered by integration tests containing the `n8n_test_` prefix.